tests/bgp/test_bgp_dual_asn.py

```diff
@@ -11,7 +11,7 @@
 from tests.common.utilities import wait_tcp_connection
 from tests.common.utilities import wait_until
 from tests.common.helpers.assertions import pytest_assert
-from bgp_helpers import update_routes
+from tests.common2.bgp_route_control import update_routes as common_update_routes
```

Comment:
The function in the common2 folder should present common one,  could we consider using common_update_routes directly as the function name in bgp_route_control? thanks.